File: JS/Sequencial/EX2_A.js


/* 2.Faca um programa que receba tres notas, calcule e mostre a media aritmetica entre elas. */


alert("Quer calcular a media aritimetica das ultimas 3 notas ?")

var med1 = prompt("Digite a primeira média")
var med2 = prompt("Digite a segunda média")
var med3 = prompt("Digite a terceira média")

med1 = parseFloat(med1)
med2 = parseFloat(med2)
med3 = parseFloat(med3)

var res = (med1 + med2 + med3) / 3;

res = parseFloat(res)

console.log("o calculo das tres medias são: "+ res.toPrecision(2))

File: Java/Algoritimos/Sequencial/EX2_A.java
package Sequencial;
import java.util.*;
import java.text.*;

/*
 * 2.Faca um programa que receba tres notas, calcule e mostre a media aritmetica entre elas.
 * */

public class EX2_A {
	
	public static void main(String args[]){
		
		float med1, med2, med3, res;
		
		Scanner entrada;
		entrada = new Scanner(System.in);
		DecimalFormat casas;
	    casas = new DecimalFormat ("0.00"); 
		
		System.out.println("Qual foi a primeira media? ");
		med1 = entrada.nextFloat();
		System.out.println("Qual foi a segunda media? ");
		med2 = entrada.nextFloat();
		System.out.println("Qual foi a terceira media? ");
	    med3 = entrada.nextFloat();
	    
	    res = (med1 + med2 + med3)/3;
	    
	    System.out.println("A média aritmetrica é: " + casas.format(res));
	    
	    entrada.close();
		
	}
}
